src/components/util/feederType.js

```diff
@@ -0,0 +1,36 @@
+import { EQUIPMENT_TYPES } from './equipment-types';
+
+const equipmentTypesWithTypeOnly = Object.fromEntries(
+    Object.entries(EQUIPMENT_TYPES).map(([key, value]) => [
+        key,
+        { type: value.type },
+    ])
+);
+
+export const FEEDER_TYPES = {
+    ...equipmentTypesWithTypeOnly,
+    /**
+     * these are ComponentTypeNames from powsybl-single-line-diagram but we put them here because some ComponentTypeNames are also EquipmentsNames
+     */
+    BUSBAR_SECTION: {
+        type: 'BUSBAR_SECTION',
+    },
+    BUS: {
+        type: 'BUS',
+    },
```

Comment:
BUSBAR_SECTION and BUS are not feeders ...

Reply:
fixed